bats/tests/helpers/load.bash

```diff
@@ -71,9 +71,6 @@ source "$PATH_BATS_HELPERS/profile.bash"
 # rdctl from commands.bash, and jq_output from utils.bash
 source "$PATH_BATS_HELPERS/vm.bash"
 
-# Use Linux utilities (like jq) on WSL
```

Comment:
I think your additions should be moved into `load.bash` instead of moving this into `paths.bash`. But I can see how you can argue it either way.